lib/cluster/storages_test.go

```diff
@@ -0,0 +1,30 @@
+package cluster_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/lib/cluster"
+)
+
+func TestStorage_GetStorageKey(t *testing.T) {
+	cases := []struct {
+		source string
+		key    string
+		err    string
+	}{
+		{"/foo/config/all", "all", ""},
+		{"/bar/config/all", "", "source must begin with: /foo/config/"},
+	}
+	for _, cs := range cases {
+		t.Run(cs.source, func(t *testing.T) {
+			key, err := cluster.GetStorageKey("/foo", cs.source)
```

Comment:
It would be nice to have tests for prefixes: /foo/, /foo/// too.
```suggestion
			key, err := cluster.GetStorageKey("/foo", cs.source)
```

Reply:
Added!